src/components/avatar/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+
+import * as Icon from '../icon';
+
+import { Image } from '../image';
+import { AvatarProps } from './types';
+
+const string: string = 'data:image/jpeg;base64,';
+
+const Avatar = ({ width = 30, height = 28, source }: AvatarProps) => {
+  const [image, setImage] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    if (!source?.includes(string) && source) {
+      setImage(`${string}${source}`);
+    }
+
+    if (source?.includes(string) && source) {
+      setImage(source);
+    }
+
+    if (!source) setImage(undefined);
+  }, [source]);
```

Comment:
I think we need to remove useEffect from this file and past in Avatar.stories.ts